homeassistant/components/mobile_app/webhook.py

```diff
@@ -78,6 +78,12 @@ def setup_registration(hass: HomeAssistantType, store: Store,
         _LOGGER.warning('Received invalid JSON from mobile_app')
         return empty_okay_response(status=HTTP_BAD_REQUEST)
 
+    if (req_data.get(ATTR_WEBHOOK_ENCRYPTED, False) is False and
```

Comment:
Don't compare to `is False`, test for key instead: `if ATTR_WEBHOOK_ENCRYPTED in req_data`